mllib/src/main/scala/org/apache/spark/ml/evaluation/MultilabelClassificationEvaluator.scala

```diff
@@ -0,0 +1,124 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.spark.ml.evaluation
+
+import org.apache.spark.annotation.{Experimental, Since}
+import org.apache.spark.ml.param.{Param, ParamMap, ParamValidators}
+import org.apache.spark.ml.param.shared.{HasLabelCol, HasPredictionCol}
+import org.apache.spark.ml.util.{DefaultParamsReadable, DefaultParamsWritable, Identifiable}
+import org.apache.spark.mllib.evaluation.MultilabelMetrics
+import org.apache.spark.sql.{DataFrame, Row}
+
+/**
+  * :: Experimental ::
+  * Evaluator for Multilabel classification, which expects two input columns: score and label.
+  */
+
+@Since("1.5.0")
+@Experimental
+class MultilabelClassificationEvaluator @Since("1.5.0") (@Since("1.5.0") override val uid: String)
+  extends HasPredictionCol with HasLabelCol with DefaultParamsWritable {
+  @Since("1.5.0")
+  def this() = this(Identifiable.randomUID("mlcEval"))
+
+  /**
+    * param for metric name in evaluation "f1Measure", "precision",
```

Comment:
Nit: each line of scaladoc needs a `*`, and we just use javadoc style where the `*` doesn't have an extra indent